ipaserver/install/bindinstance.py

```diff
@@ -423,17 +423,15 @@ def zonemgr_callback(option, opt_str, value, parser):
             encoding = getattr(sys.stdin, 'encoding', None)
             if encoding is None:
                 encoding = 'utf-8'
-            value = value.decode(encoding)
+            if not isinstance(value, unicode):
```

Comment:
Good!
I'd go just with the comment: `# value is of a string type in both py2 and py3`